src/overlays/actors/ovl_En_Fsn/z_en_fsn.c

```diff
@@ -579,8 +602,9 @@ void EnFsn_UpdateCursorAnim(EnFsn* this) {
 void EnFsn_UpdateStickDirectionPromptAnim(EnFsn* this) {
     f32 arrowAnimTween = this->arrowAnimTween;
     f32 stickAnimTween = this->stickAnimTween;
+
+    // POSSIBLY FAKE
     s32 maxColor = 255;
```

Comment:
I think it would be better to put the comment at the side instead of above
```suggestion
    s32 maxColor = 255; // POSSIBLY FAKE
```

Reply:
Applied to all four shopkeepers